the_ark/selenium_helpers.py

```diff
@@ -37,7 +37,10 @@ def create_driver(self, **desired_capabilities):
             elif desired_capabilities.get("mobile"):
                 self.driver = webdriver.Remote(desired_capabilities=desired_capabilities)
             elif desired_capabilities.get("browserName").lower() == "chrome":
-                self.driver = webdriver.Chrome()
+                options = webdriver.ChromeOptions()
+                if desired_capabilities.get("headless"):
+                    options.add_argument("headless")
+                self.driver = webdriver.Chrome(desired_capabilities=desired_capabilities, executable_path=desired_capabilities.get("binary", None), chrome_options=options)
```

Comment:
This is violating PEP8 line length, if we wanted to correct that and adhere to those standards.